views.py

```diff
@@ -299,7 +300,7 @@ def allEventsViews(page: ft.Page, params: Params, basket: Basket):
     events = Events.select()
 
     return ft.View(
-        "/events/",
+        "/event/",
```

Comment:
Лучше используй множественное число
```suggestions
"/events/",
```